dandiapi/api/views/dandiset.py

```diff
@@ -298,5 +298,7 @@ def get_user_or_400(username):
                 owners.append(owner_dict)
             except SocialAccount.DoesNotExist:
                 # Just in case some users aren't using social accounts, have a fallback
-                owners.append({'username': owner.username})
+                owners.append(
+                    {'username': owner.username, 'name': f'{owner.first_name} {owner.last_name}'}
```

Comment:
won't this just be `' '` if the user doesn't have a first/last name? I think test users always will, but I think I had to set my dev user first/last name manually. Perhaps a check that first_name or last_name are defined first?

Reply:
Since #590 users are prompted to enter their first/last name if there isn't one from github, so they will always have a name. This would include any user created when running in a dev environment.